### PR TITLE
Update BGPMON_V6 tests for both passive and active bgpmon_v6

### PR DESCRIPTION
**Pull request by Javier-Tan**
### Description of PR

Summary:
- Generalises BGPMON_V6 test to work with both active and pasive implementation of BGPMON_V6
- Add skip for known issue

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Prevent failures where in bgp/test_bgpmon_v6.py where bgpmon_v6 may be passive
#### How did you do it?
Removed check for SYN packet sent from DUT and use of information from that SYN packet following
#### How did you verify/test it?
Tested with passive bgpmon_v6 version on physical T2 device
#### Any platform specific information?
N/A
#### Supported testbed topology if it's a new test case?
N/A

### CODE REVIEW

**Comment by yejianquan** [comment]
Hi @arlakshm  @tjchadaga , could you help to review this PR?

**Comment by yejianquan** [comment]
@abdosi  could you help to review?

**Review by tjchadaga — APPROVED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -178,6 +178,10 @@ bgp/test_bgpmon.py:
     conditions:
       - "'backend' in topo_name or 't2' in topo_name"
 
+bgp/test_bgpmon_v6.py::test_bgpmon_no_ipv6_resolve_via_default:
+  skip:
+    reason: "Known issue with no ipv6 nht resolve-via-default not preventing connection to BGP neighbor"
+
 bgp/test_traffic_shift.py::test_load_minigraph_with_traffic_shift_away:
```
[the remark this answers is not in the document]
Comment (Javier-Tan) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:185]: Hi @arlakshm, that change is already there and not affected by this PR

**Review by arlakshm — DISMISSED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -178,6 +178,10 @@ bgp/test_bgpmon.py:
     conditions:
       - "'backend' in topo_name or 't2' in topo_name"
 
+bgp/test_bgpmon_v6.py::test_bgpmon_no_ipv6_resolve_via_default:
+  skip:
+    reason: "Known issue with no ipv6 nht resolve-via-default not preventing connection to BGP neighbor"
+
```
Comment (abdosi) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:184]: can we update comment that in passive mode this test is not applicable

Comment (Javier-Tan) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:184]: Updated

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**